pkg/hubble/parser/options/options.go

```diff
@@ -61,6 +62,16 @@ func Redact(logger *slog.Logger, httpQuery, httpUserInfo, kafkaApiKey bool, allo
 	}
 }
 
+// EnableL3L4PolicyEnrichment configures the Network Policy enrichment of Hubble Flows.
+func WithL3L4PolicyEnrichment(logger *slog.Logger, enableL3L4 bool) Option {
+	return func(opt *Options) {
+		opt.EnableL3L4PolicyEnrichment = enableL3L4
+		logger.Info("configured Hubble with L3/L4 network policy enrichment options",
+			logfields.Options, opt,
```

Comment:
See my next related comments
```suggestion
		logger.Info("configured Hubble with L3/L4 network policy enrichment")
```

Reply:
Updated also. I did leave the value of the bool in the log however. I think that makes sense.